bin/uat_delete_db

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+GIT_MESSAGE=$(git log --format=%B -n 1 $CIRCLE_SHA1)
+
+echo "git message is: $GIT_MESSAGE"
+
+case "$GIT_MESSAGE" in
+  "Merge pull request #"*)
+    MERGED_BRANCH=$(echo $GIT_MESSAGE | sed -n "s/^.*from ministryofjustice\/\s*\(\S*\).*$/\1/p")
+    UAT_RELEASE="$(echo $MERGED_BRANCH | sed 's:^\w*\/::' | tr -s ' _/[]().' '-' | cut -c1-30 | sed 's/-$//')"
+
+    echo "Attempting to delete UAT database for: $UAT_RELEASE"
+    HOST=$(kubectl -n ${K8S_NAMESPACE} get secret rds-instance-output -o jsonpath="{.data.rds_instance_address}" | base64 --decode)
```

Comment:
Curious to know how K8S_NAMESPACE is set and is this the variable that decides whether or not this file deletes the UAT db?

Reply:
`K8S_NAMESPACE` is set in the CIRCLE CI context, so it will be specific to the environment, e.g. ends with `-uat` in this instance, to ensure we don't accidentally delete `-prod` databases 😄 

This allows us to connect to the UAT RDS instance and issue the instructions